driverkit/Makefile

```diff
@@ -31,7 +31,7 @@ define gen_build_targets
 $(1): $(3)
 	utils/checkfiles $(3)
 	@mkdir -p output/$(2)
-	@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 || echo $(3) >> output/failing.log
+	@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 && echo $(3) >> output/failing.log
```

Comment:
I only have a couple of concerns:
- we're fulfilling the `failing` log with `successful` builds
- as drivers build can be time-expensive, we could have a time-expensive trial & error if we exit on first failure
- I'd keep DBG and build jobs separate as DBG can be used also locally (for example maybe I'd want to build locally all the drivers and see only at the end the result avoiding trial and error)

My proposals below.

#### Control inside DBG

if we want to exit on first failure, in `driverkit/Makefile`:

```
@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 || { echo $(3) >> output/failing.log && exit 1 ; }
```
#### Control inside build job

If we don't change the DBG and check after DBG run complete, in `images/build-drivers/build-drivers.sh`:

```
$make $DBG_MAKE_BUILD_OPTIONS $DBG_MAKE_BUILD_TARGET

test -s output/failing.log || { pretty_echo "DBG build failed"  && cat output/failing.log && exit 1 ; }

pretty_echo "DBG build complete"
```

Reply:
I don't think this is a viable solution.

That `or` operator was meant to make the Makefile write the failing builds to `output/failing.log`

If the problem is making it exit with status != 0 in that case .. Well just use `exit 1` on that conditional branch :)

Reply:
```suggestion
	@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 && { echo $(3) >> output/failing.log; exit 1; }
```

Reply:
Maybe the first `&&` should remain an or `||`, right @leodido?